### PR TITLE
Cura: update to 20211110

### PR DESCRIPTION
**Pull request by ryanfortner**

### CODE REVIEW

**Comment by Itai-Nelken** [comment]
Could you add a version variable and use that? It will make updatig easier next time.

**Comment by ryanfortner** [comment]
> Could you add a version variable and use that? It will make updatig easier next time.

Added.

File: apps/Cura/install-32 [hunk 1]
```diff
@@ -7,15 +7,15 @@ function error {
...
+wget -O ~/Cura.AppImage https://github.com/smartavionics/Cura/releases/download/${version}/Cura-mb-master-armhf-${version}.AppImage || error "Failed to download!"
 
 chmod +x ~/Cura.AppImage
 
 #get stl-thumb package
-rm -f ~/stl-thumb_0.4.0_armhf.deb
-wget https://github.com/unlimitedbacon/stl-thumb/releases/download/v0.4.0/stl-thumb_0.4.0_armhf.deb
-"${DIRECTORY}/pkg-install" "$HOME/stl-thumb_0.4.0_armhf.deb" "$(dirname "$0")" || exit 1
-rm -f ~/stl-thumb_0.4.0_armhf.deb
+stl_thumb_version='0.4.0'
```
Comment (Itai-Nelken) [inline apps/Cura/install-32:17]: Thanks! Maybe put all the variables at the top?

**Comment by ryanfortner** [comment]
Done. Ready for merge?

**Comment by Itai-Nelken** [comment]
Yes

**Pull request merged**